src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -808,7 +857,7 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, Component
                     translateValues: { index: index + 1 },
                 });
             }
-            if (question.score && question.score < 0) {
+            if (question.score === undefined || question.score === null || question.score < 0) {
```

Comment:
Are these undefined checks really necessary? Wasn't `question.score && ...` already correct?

Reply:
in case the score is null or undefined, the condition would be false and not triggered. But we want it to be triggered in those cases.